cluster-scope/base/core/namespaces/smart-village-view/resourcequota.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: opf-ci-pipelines-custom
```

Nit, please change the name to match the namespace name.

```suggestion
  name: smart-village-view-custom
```

---

Great feedback @tumido , I have renamed my resourcequota.